Cargo.toml

```diff
@@ -35,6 +39,7 @@ tar = "0.4.16"
 toml = "0.4"
 which = "2.0.0"
 zip = "0.4.2"
```

Comment:
I think some crates in this list (like `zip` here) may no longer be necessary, mind pruning this of unneeded crates?

Reply:
Ah, my bad. Will fix that!